packages/mui-base/src/useMenuItem/useMenuItem.ts

```diff
@@ -55,6 +55,7 @@ export function useMenuItem(params: UseMenuItemParameters): UseMenuItemReturnVal
 
   const { getRootProps: getListRootProps, highlighted } = useListItem({
     item: id,
+    handlePointerOverEvents: true,
```

Comment:
We should add a parameter to `useMenuItem` to control this so the behavior can be modified. This is required for Material that has different hover and focus states.

Reply:
Added.